src/main/java/com/stories/StoriesApplication.java

```diff
@@ -0,0 +1,16 @@
+package com.stories;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+@SpringBootApplication
+@ComponentScan("com.stories")
```

Comment:
There's no need to add the package inside the @ComponentScan since it's the same as the package where this class is in.